File: .github/src/model/Card.py
from model.common.WalletObject import WalletObject


class Card(WalletObject):
    def __init__(self, id, card_type):
        super().__init__(id)
        self._type = card_type

    def get_type(self):
        return self._type

    def __str__(self):
        return f"Card{{type={self._type}}}"

File: .github/src/model/Card.dart
import 'wallet_object.dart';
import 'card_type.dart';

class Card extends WalletObject {
  final CardType type;

  Card(int id, this.type) : super(id);

  CardType getType() {
    return type;
  }

  @override
  String toString() {
    return 'Card{type=$type}';
  }
}
